src/enums.py

```diff
@@ -211,3 +211,25 @@ class Direction(StrEnum):
 class EntityState(StrEnum):
     IDLE = "idle"
     WALK = "walk"
+
+
+class LAYER(IntEnum):
```

Comment:
This is Nitpicking: Extreme Edition™️, but all caps names are preferably used for constants only, so this class name should have been in PascalCase (Layer)

Reply:
No problem
Thank you very much 😄